core/src/test/java/org/infinispan/xsite/offline/AsyncOfflineTest.java

```diff
@@ -116,13 +120,23 @@ public void testReset(Method method) {
       List<DiscardInboundHandler> handlers = replaceSFOInboundHandler();
       handlers.forEach(h -> h.discard = true);
 
-      //SFO request should timeout
+      //SFO request should timeout (timeout=1 sec) and it retries (max retries = 6)
```

Comment:
Shouldn't the test configure a lower number of retries to make things faster?

Reply:
no. the test only waits for the first failure and it should never be reached